invoicer_project/static/js/add_client_request.js

```diff
@@ -0,0 +1,286 @@
+const nameSurnMaxLength = 35;
+const countryMaxLength = 35;
+const addressMaxLength = 40;
+const host = "http://127.0.0.1:8000";
+
+const nameField = document.getElementById("name_input_client_edit_page");
+const surnameField = document.getElementById("surname_input_client_edit_page");
+const emailField = document.getElementById("email_input_client_edit_page");
+const telephoneField = document.getElementById("telephone_input_client_edit_page");
+const zipField = document.getElementById("zip_input_client_edit_page");
+const countryField = document.getElementById("country_input_client_edit_page");
+const cityField = document.getElementById("city_input_client_edit_page");
+const addressField = document.getElementById("address_input_client_edit_page");
+
+const returnAllFields = function () {
+    return [
+        nameField, surnameField,
+        emailField, telephoneField,
+        zipField, countryField,
+        cityField, addressField
+    ];
+}
+
+for (let field of returnAllFields()) {
```

Comment:
let's introduce the method `cleareErrorAttributes` and movet this loop inside this method

Reply:
corrected